### DISCUSSION TITLE
"- Nextra" automatically added to the page title?

### DISCUSSION THREAD

**Discussion by nshiab** [shuding/nextra#2831] [Q&A] [answered]
Hi!

First, thank you very much for this great project.

I noticed that " - Nextra" is automatically added to my page title.

Is there a way to avoid that?

Thanks!

**Comment by caffeine-addictt** [comment, marked as answer]
What does your `theme.config.tsx` look like?

If it looks something like this, consider removing the "- Nextra"
```tsx
import type { DocsThemeConfig } from 'nextra-theme-docs'

const config: DocsThemeConfig = {
  ...,
  useNextSeoProps() {
    const { asPath } = useRouter()
    if (asPath !== '/') {
      return {
        titleTemplate: '%s – Nextra'
      }
    }
  },
  ...
}

export defualt config
```

**Reply by nshiab** [reply]
Ah! I wasn't using useNextSeoProps. This works. Thank you very much!

**Reply by cyrildiagne** [reply]
> What does your `theme.config.tsx` look like?
> 
> If it looks something like this, consider removing the "- Nextra"
> 
> ```tsx
> import type { DocsThemeConfig } from 'nextra-theme-docs'
> 
> const config: DocsThemeConfig = {
>   ...,
>   useNextSeoProps() {
>     const { asPath } = useRouter()
>     if (asPath !== '/') {
>       return {
>         titleTemplate: '%s – Nextra'
>       }
>     }
>   },
>   ...
> }
> 
> export defualt config
> ```


It looks like `useNextSeoProps` is no longer available. The doc seems to suggest that we need to return a full head component to remove the "- Nextra"?

**Comment by dimaMachina** [comment]
`useNextSeoProps` was removed in Nextra 3 https://the-guild.dev/blog/nextra-3#removes-of-several-themeconfig-options-eslint-disable-line-

**Reply by hoshank** [reply]
@dimaMachina or someone here can show us how to modify the Nextra word being appended to the title of every page ? what would the theme.config.tsx look like ?

**Reply by GermanJablo** [reply]
@dimaMachina 

I've been trying for a while to migrate from v2 to v3 without success. Could you please explain how I can change the titles of my pages and still use my settings (logo, sidebar, etc.).

My current `theme.config.tsx` looks like this:

```jsx
import Logo from "./public/logo";

const config = {
  docsRepositoryBase: "https://github.com/xxx/xxx/docs",
  logo: (<Logo />),
  project: {
    link: "https://github.com/xxx/xxx",
  },
  useNextSeoProps() {
    return {
      titleTemplate: "%s - xxx",
    };
  },
  sidebar: {
    defaultMenuCollapseLevel: 1,
  },
  footer: false,
  chat: {
    link: "https://discord.gg/xxx",
  },
};

export default config;

```


Thank you!

**Reply by dimaMachina** [reply]
Did you read my previous comment? `useNextSeoProps` was removed, use `head` option
